.github/workflows/test.yml

```diff
@@ -22,8 +22,23 @@ jobs:
       with:
         node-version: '14.x'
     - uses: bahmutov/npm-install@v1
+      with:
+        install-command: yarn --frozen-lockfile --non-interactive --silent
     - name: Build
       run: make build
+
+  unit-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js LTS
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - uses: bahmutov/npm-install@v1
+      with:
+        install-command: yarn --frozen-lockfile --non-interactive --silent
     - name: Lint
```

Comment:
shouldn't we put that in the typecheck phase, too?

Reply:
It's more about having all jobs taking approximately the same time

Reply:
unit-test is already the fastest job so it make sense to keep the linting there